src/tfont/converters/ufoConverter.py

```diff
@@ -104,68 +107,89 @@ def open(self, path, font=None):
         # glyphs
         font._glyphs.clear()
         glyphs = font.glyphs
-        for g in ufo:
-            glyph = Glyph(g.name)
+        for glyph_name in ufo.glyphOrder:
```

Comment:
glyphOrder isn't guaranteed to be set or to contain all glyphs in the font, or to contain only glyphs that exist in the font

Reply:
Correct

Reply:
That's why it's first processed in the old trufont: https://github.com/trufont/trufont/blob/b5d48887aaacbfd628ae30d2c102252f37a931cd/Lib/trufont/windows/fontWindow.py#L518-L533

Reply:
Oops, right. Time to split off stuff from `open()` with an eye towards designspace loading :)

Reply:
: ) I'll look at your other PR also.